### PR TITLE
#26 api Services Classes Done

### PR DESCRIPTION
**Pull request by Jallow9**
issue #26 Api Services Classes,

### CODE REVIEW

File: frontend/challedu-app/src/UserProvider.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+import APIUtils from "./APIUtils"
+
+export default class UserProvider {
+  static async getUsers() {
+    APIUtils.get('users/')
```
Comment (Pauleing) [inline frontend/challedu-app/src/UserProvider.js:6]: missing retur

File: frontend/challedu-app/src/UserProvider.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  }
+  static async getUser(id) {
+    APIUtils.get(`users/${id}`);
+  }
+  static async createUser(userData) {
+    APIUtils.post('users/', userData );
```
Comment (Pauleing) [inline frontend/challedu-app/src/UserProvider.js:12]: here alsoooo

File: frontend/challedu-app/src/APIUtils.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+
+// this is our server and port address
+// the base of all AJAX requests
+
+const API_URL= 'http://localhost:4321/';
+
+export default class APIUtils {
+  static async get(url) {
+    result await issueAJAXRequest('GET',url);
```
Comment (Pauleing) [inline frontend/challedu-app/src/APIUtils.js:9]: here is return!!!

**Comment by Manolispro** [comment]
APIUtils.js lines 27 28 29 33 change the res to result

**Pull request merged**